janus_core/cli/md.py

```diff
@@ -375,6 +378,10 @@ def md(
         ]
     )
 
+    # Handle separately to process short-hands, and Observables.
+    if correlation_kwargs:
+        correlation_kwargs = parse_correlation_kwargs(correlation_kwargs)
```

Comment:
For the summary, I think you need to add a copy of `correlation_kwargs to `all_kwargs`, ideally before this parsing if possible. The idea is `all_kwargs` makes a copy of the commands you actually passed, before any processing etc.